fjord/analytics/analyzer_views.py

```diff
@@ -251,57 +253,61 @@ def unknown_to_empty(text):
         return u'' if text.lower() == u'unknown' else text
 
     if search_platform is not None:
-        f &= F(platform=unknown_to_empty(search_platform))
+        f &= F('term', platform=unknown_to_empty(search_platform))
         current_search['platform'] = search_platform
     if search_locale is not None:
-        f &= F(locale=unknown_to_empty(search_locale))
+        f &= F('term', locale=unknown_to_empty(search_locale))
         current_search['locale'] = search_locale
     if search_product is not None:
-        f &= F(product=unknown_to_empty(search_product))
+        f &= F('term', product=unknown_to_empty(search_product))
         current_search['product'] = search_product
 
         # Only show the version if there's a product.
         if search_version is not None:
             # Note: We only filter on version if we're filtering on
             # product.
-            f &= F(version=unknown_to_empty(search_version))
+            f &= F('term', version=unknown_to_empty(search_version))
             current_search['version'] = search_version
 
         # Only show the country if the product is Firefox OS.
         if search_country is not None and search_product == 'Firefox OS':
-            f &= F(country=unknown_to_empty(search_country))
+            f &= F('term', country=unknown_to_empty(search_country))
             current_search['country'] = search_country
     if search_domain is not None:
-        f &= F(url_domain=unknown_to_empty(search_domain))
+        f &= F('term', url_domain=unknown_to_empty(search_domain))
         current_search['domain'] = search_domain
     if search_api is not None:
-        f &= F(api=search_api)
+        f &= F('term', api=search_api)
         current_search['api'] = search_api
 
     if search_date_start is None and search_date_end is None:
         selected = '7d'
 
     if search_date_end is None:
-        search_date_end = datetime.now()
+        search_date_end = date.today()
```

Comment:
This was a bug. It should never have been a datetime since we're only looking at dates and not time-of-days.